### ISSUE TITLE
add zoneminder rock-on guide

### ISSUE THREAD

**Issue by phillxnet** [rockstor/rockstor-doc#152]
Given zoneminders popularity as a security camera management system I think it would be good to have a dedicated Rock-on howto. This could also cover the initial setup of zoneminder for a simple single camera install.

I am in the process of setting up a Zoneminder Rockstor install so will attempt to prepare this howto as I go. Any additional notes here on suggested share sizes etc are most welcome.

**Comment by schakrava** [comment]
As time permits, I'd really like to try this rock-on at home myself.

**Comment by phillxnet** [comment]
@schakrava Yes, it's a great addition. When I last set it up for a client some years ago I used an android app to act as an ip camera for testing purposes. I'll try and put this in the write-up as an example camera setup and explain the ffmpeg settings once I've sussed them.

Have done most screen grabs for install and initial config stages. Will attempt to get first draft done shortly, hopefully in time for you to test run the guide. My original zoneminder server (Ubuntu sever 11.04 based) private notes are 17 pages long without pics!!
